### PR TITLE
Patch for lvsetup.py to work for Ubuntu Distro

### PR DESCRIPTION
**Pull request by hathyaga**
xfs and btrfs packages added to work for Ubuntu Distro

Signed-off-by: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>

### CODE REVIEW

File: io/disk/lvsetup.py [hunk 1]
```diff
@@ -45,6 +47,11 @@ def setUp(self):
         """
         Check existence of input PV,VG, LV and snapshots prior to Test.
         """
+        smm = SoftwareManager()
+        pkgs = ['xfsprogs', 'btrfs-progs']
+        for pkg in pkgs:
+            if not smm.check_installed(pkg) and not smm.install(pkg):
+                self.skip("%s package is need to test" % pkg)
```
Comment (narasimhan-v) [inline io/disk/lvsetup.py]: You need not do it always. Install these packages in case of xfs/btrfs alone.

**Comment by PraveenPenguin** [comment]
@hathyaga please rebase as remove  `distro import` as it not used , it will make Travis happy :)

**Comment by narasimhan-v** [comment]
@hathyaga Also, the packages are generic, but commit message says it is ubuntu. Please change them accordingly

File: io/disk/lvsetup.py (hunk 1 above)
Comment (hathyaga) [inline io/disk/lvsetup.py]: These packages are not required for Rhel. So can I do a distro.detect and install these for Ubuntu OS?

Comment (narasimhan-v) [inline io/disk/lvsetup.py]: yes, if these are the packages for rhel, better do it for rhel as well :)

**Comment by hathyaga** [comment]
@PraveenPenguin @narasimhan-v 
New PR: https://github.com/avocado-framework-tests/avocado-misc-tests/pull/282

**Comment by hathyaga** [comment]
Closing this PR.

**Pull request closed without merging**